DQM/BeamMonitor/plugins/BeamConditionsMonitor.h

```diff
@@ -11,49 +11,64 @@
 #include <string>
 // CMS
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "DQMServices/Core/interface/DQMGlobalEDAnalyzer.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "DQMServices/Core/interface/ConcurrentMonitorElement.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "CondFormats/BeamSpotObjects/interface/BeamSpotObjects.h"
 
 //
 // class declaration
 //
-namespace beamcond {
-  struct RunCache {
-    typedef dqm::reco::MonitorElement MonitorElement;
-    // MonitorElements
-    ConcurrentMonitorElement h_x0_lumi;
-    ConcurrentMonitorElement h_y0_lumi;
-  };
-};  // namespace beamcond
-
-class BeamConditionsMonitor : public DQMGlobalEDAnalyzer<beamcond::RunCache, edm::LuminosityBlockCache<void>> {
+
+class BeamConditionsMonitor : public edm::EDAnalyzer {
```

Comment:
Isn't the use of `edm::EDAnalyzer` deprecated since years ?

Reply:
It is, I just reverted this module back to it's previous state becaue I was not really sure what the correct semantics in multi-threaded running would be.

It's a rather pointless exercise to try to fix it, it is only supposed to run online anyways. And actually, as far as I see, it is not running at all anywhere right now. I might just delete it entirely.